src/backend/base/pyproject.toml

```diff
@@ -211,34 +211,258 @@ pytest-cmd = "pytest -p no:profiling -p no:sugar -p no:xdist -p no:cov -p no:spl
+mongodb-vectors = ["langchain-mongodb==0.2.0"]
+astradb = ["langchain-astradb==0.5.2"]
+
+# Individual LLM providers
+openai = ["langchain-openai==0.2.12"]
+anthropic = ["langchain-anthropic==0.3.0"]
+cohere = ["langchain-cohere==0.3.3"]
+google = [
+    "langchain-google-genai==2.0.6",
+    "langchain-google-vertexai==2.0.7",
+    "langchain-google-community==2.0.3",
+    "langchain-google-calendar-tools==0.0.1"
+]
+ollama = ["langchain-ollama==0.2.1"]
+nvidia = ["langchain-nvidia-ai-endpoints==0.3.8"]
+mistral = ["langchain-mistralai==0.2.3"]
+sambanova = ["langchain-sambanova==0.1.0"]
+groq = ["langchain-groq==0.2.1"]
+
+# Individual local LLM providers
+llama-cpp = ["llama-cpp-python~=0.2.0"]
+sentence-transformers = ["sentence-transformers>=2.3.1"]
+ctransformers = ["ctransformers>=0.2.10"]
+
+# Individual monitoring providers
+langfuse = ["langfuse==2.53.9"]
+langwatch = ["langwatch==0.1.16"]
+langsmith = ["langsmith==0.1.147"]
+arize = ["arize-phoenix-otel>=0.6.1"]
+
+# Individual document loaders
+pypdf = ["pypdf~=5.1.0"]
+docx = ["python-docx>=1.1.0"]
+pytube = ["pytube==15.0.0"]
+
+# Individual agent frameworks
+crewai = ["crewai==0.102.0"]
+dspy = ["dspy-ai==2.5.41"]
+smolagents = ["smolagents>=1.8.0"]
+
+# Individual tools
+beautifulsoup = ["beautifulsoup4==4.12.3"]
+serpapi = ["google-search-results>=2.4.1,<3.0.0"]
+google-api = ["google-api-python-client==2.154.0"]
+wikipedia = ["wikipedia==1.4.0"]
+fake-useragent = ["fake-useragent==1.5.1"]
+duckduckgo = ["duckduckgo_search==7.2.1"]
+yfinance = ["yfinance==0.2.50"]
+wolframalpha = ["wolframalpha==5.1.3"]
+
+# Individual utilities
+pyarrow = ["pyarrow==19.0.0"]
+fastavro = ["fastavro==1.9.7"]
+gitpython = ["GitPython==3.1.43"]
+nltk = ["nltk==3.9.1"]
+lark = ["lark==1.2.2"]
+
+# Additional dependencies from requirements
+huggingface = ["huggingface-hub[inference]>=0.23.2,<1.0.0"]
+metaphor = ["metaphor-python==0.1.23"]
+metal = ["metal_sdk==2.5.1"]
+markup = ["MarkupSafe==3.0.2"]
+aws = ["boto3==1.34.162"]
+numexpr = ["numexpr==2.10.2"]
+qianfan = ["qianfan==0.3.5"]
+pgvector = ["pgvector==0.3.6"]
+litellm = ["litellm==1.60.2"]
+zep = ["zep-python==2.0.2"]
+youtube = ["youtube-transcript-api==0.6.3"]
+markdown = ["Markdown==3.7"]
+kubernetes = ["kubernetes==31.0.0"]
+json-repair = ["json_repair==0.30.3"]
+astra = ["astra-assistants[tools]~=2.2.9"]
+composio = [
+    "composio-langchain==0.7.1",
+    "composio-core==0.7.1"
 ]
+ragstack = ["ragstack-ai-knowledge-store==0.2.1"]
+opensearch = ["opensearch-py==2.8.0"]
+atlassian = ["atlassian-python-api==3.41.16"]
+mem0 = ["mem0ai==0.1.34"]
+needle = ["needle-python>=0.4.0"]
+sseclient = ["sseclient-py==1.8.0"]
+openinference = ["openinference-instrumentation-langchain>=0.1.29"]
+mcp = ["mcp>=0.9.1"]
+ag2 = ["ag2>=0.1.0"]
+scrapegraph = ["scrapegraph-py>=1.12.0"]
+pydantic-ai = ["pydantic-ai>=0.0.19"]
+apify = ["apify-client>=1.8.1"]
+spider = ["spider-client>=0.0.27,<1.0.0"]
+
+# Development dependencies
+dev = [
```

Comment:
I'm not sure there is interest in making a `dev` extra.
Shouldn't it be a group ?

Reply:
Yep. Forgot to remove. Thanks!